test/clojure/nrepl/describe_test.clj

```diff
@@ -1,18 +1,20 @@
 (ns nrepl.describe-test
   {:author "Chas Emerick"}
-  (:require
-   [clojure.test :refer :all]
-   [nrepl.core :as nrepl]
-   [nrepl.core-test :refer [def-repl-test repl-server-fixture project-base-dir]]
-   [nrepl.middleware :as middleware]
-   [nrepl.version :as version]))
+  (:require [clojure.test :refer [is testing use-fixtures]]
```

Comment:
I think you reformatted this accidentally - all the code sticks to the other style.

Reply:
do you mean the `clojure.test :refer :all` part or the first spec on the line after `(:require .. `part?

Reply:
I mean having like-break after `:require`.

Reply:
Reverted.